src/pages/blog/videos/_video-card.js

```diff
@@ -0,0 +1,161 @@
+    overflow: hidden;
+
+    @media ${device.mobileL} {
+        width: 100%;
+    }
+`
+
+const ImageOverlay = styled.div`
+    position: absolute;
+    top: 0;
+    left: 0;
+    z-index: 2;
+    width: 100%;
+    height: 100%;
+    background-color: rgba(14, 14, 14, 0.5);
+`
+
+const CategoriesContainer = styled(Flex)`
+    position: absolute;
+    align-items: flex-start;
+    justify-content: flex-start;
+    height: fit-content;
+    padding: 8px;
+    z-index: 3;
+`
+
+const StyledCategories = styled(Header)`
+    width: fit-content;
+    padding: 2px 8px 0;
+    border-radius: 8px;
+    background-color: var(--color-brown);
+    color: var(--color-orange-2);
+    margin: 0 8px 8px 0;
+`
+
+const PlayButtonOval = styled.div`
+    position: absolute;
+    z-index: 2;
+    top: 50%;
+    left: 50%;
+    transform: translate(-50%, -50%);
+    height: 80px;
+    width: 80px;
+    background-color: var(--color-white);
+    opacity: 0.32;
+    border-radius: 50%;
+`
+
+const StyledTriangle = styled.img`
+    position: absolute;
+    z-index: 4;
+    top: 50%;
+    left: 50%;
+    transform: translate(-40%, -50%);
+`
+
+const VideoDuration = styled(Header)`
+    width: auto;
+    position: absolute;
+    bottom: 8px;
+    right: 8px;
+    z-index: 3;
+    border-radius: 3px;
+    background-color: var(--color-black);
+    color: var(--color-white);
+    padding: 0 4px;
+`
+
+const ContentWrapper = styled.div`
+    padding: 16px 24px;
+`
+
+const VideoCard = ({ item, openVideo }) => {
+    return (
+        // the extra div surrounding the videocard is to get around Safari's different
+        // interpretation of height: 100%
+        <div>
+            <VideoCardWrapper onClick={openVideo}>
+                <ImageWrapper>
+                    <ImageOverlay />
+                    <CategoriesContainer jc="flex-start" fw="wrap">
+                        {item.category.slice(0, 2).map((item_category) => (
+                            <StyledCategories as="h4" type="paragraph-2" key={item_category}>
+                                {item_category}
+                            </StyledCategories>
+                        ))}
+                        {item.category.length > 2 && (
+                            <StyledCategories as="h4" type="paragraph-2">
+                                {`+${item.category.slice(2).length.toString()}`}
+                            </StyledCategories>
+                        )}
+                    </CategoriesContainer>
+                    <PlayButtonOval />
+                    <StyledTriangle src={Triangle} alt="play button" height={25} width={20} />
+                    <VideoDuration as="h5" type="paragraph-2" weight="bold">
+                        {item.video_duration}
+                    </VideoDuration>
+                    <img
+                        src={item.image}
+                        alt="Video card"
+                        width="100%"
+                        style={{ objectFit: 'contain' }}
```

Comment:
seems duplicated everywhere. we can make a styled component under blog/_components.js, and put Img with this style